internal/vpn/os_linux.go

```diff
@@ -42,3 +43,35 @@ func AddRoute(ip, gateway string) error {
 func DeleteRoute(ip, gateway string) error {
 	return run("ip", "r", "del", ip, "via", gateway)
 }
+
+var (
```

Comment:
Could we get rid of this block and return errors from `syscall` directly? I think this doesn't improve readability.

Reply:
Not used anymore